io.github.picocrypt.Picocrypt.yaml

```diff
@@ -0,0 +1,40 @@
+app-id: io.github.picocrypt.Picocrypt
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: picocrypt-gui
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=home
+modules:
+  - name: picocrypt-gui
+    buildsystem: simple
+    build-commands:
+      - touch io.github.picocrypt.Picocrypt.desktop
+      - echo "[Desktop Entry]" >> io.github.picocrypt.Picocrypt.desktop
+      - echo "Name=Picocrypt" >> io.github.picocrypt.Picocrypt.desktop
+      - echo "Comment=Secure file encryption tool" >> io.github.picocrypt.Picocrypt.desktop
+      - echo "Type=Application" >> io.github.picocrypt.Picocrypt.desktop
+      - echo "Exec=picocrypt-gui" >> io.github.picocrypt.Picocrypt.desktop
+      - echo "Icon=io.github.picocrypt.Picocrypt" >> io.github.picocrypt.Picocrypt.desktop
+      - echo "Categories=Utility;Security;" >> io.github.picocrypt.Picocrypt.desktop
+      - install -Dm755 Picocrypt /app/bin/picocrypt-gui
+      - install -Dm644 key.svg /app/share/icons/hicolor/scalable/apps/io.github.picocrypt.Picocrypt.svg
+      - install -Dm644 io.github.picocrypt.Picocrypt.desktop /app/share/applications/io.github.picocrypt.Picocrypt.desktop
+      - install -Dm644 io.github.picocrypt.Picocrypt.metainfo.xml /app/share/metainfo/io.github.picocrypt.Picocrypt.metainfo.xml
+    sources:
+      - type: file
+        url: https://github.com/Picocrypt/Picocrypt/releases/download/1.42/Picocrypt
+        sha256: 7112b72d0b62337f9249e47f6079b8a041c445fc089f0610ca3af040514e4ab2
+      - type: file
+        url: https://raw.githubusercontent.com/Picocrypt/Picocrypt/main/images/key.svg
+        sha256: f2f3b9ad8322d8e0d9bc3ba888fa89310bfe54800f48d53a4260c71c96f79cf6
+      - type: file
+        url: https://raw.githubusercontent.com/Picocrypt/flathub/io.github.picocrypt.Picocrypt/io.github.picocrypt.Picocrypt.metainfo.xml
```

Comment:
no. this reference the wrong place. 

1. it's like so it's not URL
2. it actually should be upstream